### DISCUSSION TITLE
SP-API Messaging API how can I get buyer Messages and response to it?

### DISCUSSION THREAD

**Discussion by romeolandry** [amzn/selling-partner-api-models#3487] [General] [answered]
Hey every body,
We wont to implement custom  communication API to Iteract with buyer a kind of (ping-pong). But I read into Messaging API that the don't allow sesnding response to Buyer or response to buyer Message. 

How can I Implement that?

**Comment by wanglei0214** [comment, marked as answer]
Hi romeolandry,

We currently do not provide a functionality to let developer get the buyer Messages and response to it.
This is only the feature to let you send a message with the specified template from Amazon.

We have aware of the needs from developers like you. 
We will communicate with the product team to request this feature.
Please keep an eye on the our doc site for new announcement if once it is featured in the future.

Best,
Lei

**Reply by romeolandry** [reply]
Thanks for Answer.

**Reply by bgraves63** [reply]
We would also like this functionality.  Sometimes a customer emails a question and we respond.  Then the customer replies with "Thanks".  We need to mark the email as No Response Required.  We would like an API so we can mark the email "No Response Required" from our CRM.  Thanks

**Reply by alixleger** [reply]
Hi @wanglei0214 , do you know if it will be featured this year ?

**Comment by lombervid** [comment]
Hello. I would like to know if this functionality has not yet been implemented?

Kind Regards!

**Comment by Bithellio** [comment]
Hi @wanglei0214, 
 Just checking in again like @lombervid  above ..  
is there any clue when this may be implemented or if this has been implemented . its a really handy option to be able to mark messages as no response needed via API 

thanks in advance

**Comment by maciuchx** [comment]
Hi @wanglei0214 do we have any update on the "No response needed" via API?

**Comment by nothinman** [comment]
come on guys, in the age of AI, not having this functionality live by now is a joke. do better!
